system-addon/content-src/components/PreferencesPane/PreferencesPane.jsx

```diff
@@ -76,6 +82,11 @@ class PreferencesPane extends React.Component {
               <PreferencesInput className="showTopSites" prefName="showTopSites" value={prefs.showTopSites} onChange={this.handleChange}
                 titleStringId="settings_pane_topsites_header" descStringId="settings_pane_topsites_body" />
 
+              <div className="options">
+                <PreferencesInput className="showMoreTopSites" prefName="topSitesCount" value={prefs.topSitesCount !== 6} onChange={this.handleChange}
```

Comment:
I would keep number of topsites. It's what we used until now and what everyone is used to.